captum/attr/_utils/attribution.py

```diff
@@ -3,13 +3,13 @@
 import torch.nn.functional as F
 
 from .common import (
-    _run_forward,
+    _format_additional_forward_args,
     _format_input_baseline,
     _format_tensor_into_tuples,
-    _format_additional_forward_args,
+    _run_forward,
+    _tensorize_baseline,
```

Comment:
nit: it looks like restructured the order of imports. Did black formatting do it ?

Reply:
Yes, I believe one of the formatting linters requested this change.